src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -64,9 +66,16 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
     }
 
     renderButtons(paypalCheckoutInstance: PaypalClientInstance) {
-        const { paypalOptions, paymentMethod, container } = this._renderButtonsData as RenderButtonsData;
+        const { paypalOptions, paymentMethod, container, messagingContainerId } = this._renderButtonsData as RenderButtonsData;
         const { paypal } = this._window;
+        const state = this._store.getState();
+        const cart = state.cart.getCartOrThrow();
         let updatedPaypalOptions: BraintreePaypalButtonInitializeOptions;
+        const isMessageContainerAvailable = Boolean(messagingContainerId && document.getElementById(messagingContainerId));
+        const ppsdkFeatureOn =
+            this._store.getState()
```

Comment:
Think @Sireks has a better suggestion above, so maybe try that.

Reply:
Fixed